docs/topics/mpp/mpp-share-on-platforms.md

```diff
@@ -111,6 +111,56 @@ kotlin {
 ```
 
 </tabs>
+
+#### Target shortcuts and ARM64 (Apple Silicon) simulators
+
+The target shortcuts `ios`, `watchos` and `tvos` don't include the simulator targets for ARM64 (Apple Silicon) platforms:
```

Comment:
```suggestion
The target shortcuts `ios`, `watchos`, and `tvos` don't include the simulator targets for ARM64 (Apple Silicon) platforms:
```

Reply:
Hmm, I first put an serial comma, but now I'm not sure if it is needed here 🤔

Reply:
I think you're right, we need one here.